src/redux/country/detailsSlice.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ]  Kindly correct your linters instead of disabling the. they help you prevent future bugs in the app and help you apply best practices.